### PR TITLE
[feat] Create Snackbar and Documentation page

### PR DESCRIPTION
**Pull request by Hamdrive**
Created two types of snackbars : Baseline and Lading
Also created the documentation page for the same

snackbar-doc

### CODE REVIEW

**Review by ZuberDunge — COMMENTED** [review]
Excellent code structure, good to go from my side.

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    align-items: center;
+    flex-wrap: wrap;
+    z-index: 2;
+    color: var(--color-white);
+}
+
+.baseline button,
+.leading button,
+.stacked button {
+  color: blueviolet;
```
Comment (stutisk) [inline components/snackbar/snackbar.css]: you can declare this color as a variable

File: components/snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  margin: 0 0.5rem;
+}
+
+.close-icon i {
+  font-size: 1.2rem;
+  color: white;
+}
+
+.close-icon:hover {
+  background-color: grey;
```
Comment (stutisk) [inline components/snackbar/snackbar.css]: here also

**Comment by stutisk** [comment]
else  overall code looks good to me

**Comment by Hamdrive** [comment]
Merging to dev

**Pull request merged**